exercises/en/test_08_12.py

```diff
@@ -0,0 +1,14 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+    import pandas as pd
+    # If an assertion fails, the message will be displayed
+    assert "make_pipeline" in __solution__, "Make sure creating a pipeline with the preprocessor."
+    assert "LogisticRegression" in __solution__, "Make sure you are specifying logistic regression in the pipeline."
```

Comment:
Can you check the pipeline has `LogisticRegression` as a step in the object instead of the string? This is because we need to make sure they are importing it too.